PartTwo.js

```diff
@@ -23,7 +26,7 @@ function zero(){
 
 
 // Part 2-1
-function one(){
+function SumAge(){
```

Comment:
functions should always start with a lowercase letter, Classes start capitalized. Helps with readability